packages/react-vapor/src/components/modal/tests/ModalComposite.spec.tsx

```diff
@@ -53,6 +54,12 @@ describe('ModalComposite', () => {
         expect(modalContent.find(ModalHeader).exists()).toBe(true);
     });
 
+    it('renders an accessible heading', () => {
+        renderModal(<ModalComposite isOpened title={'my title'} />);

```

Comment:
I think it would be worth testing with 2 modals at the same time.

```suggestion
        renderModal(
            <>
                <ModalComposite isOpened title="my title" />
                <ModalComposite isOpened title="other title" />
            </>
        );

```